### ISSUE TITLE
Timeouts

### ISSUE THREAD

**Issue by mrivi** [PX4/avoidance#118]
I get timeouts messages `Pointcloud timeout , no cloud received on topic  (Hovering at current position) ` both in the simulation and on hardware.

I have already spoken offline with @baumanta. As she suspected, the timeout happens because the transformation between the camera frame and local origin is not available.

Opening a issue so that we track.

**Comment by mrivi** [comment]
This is what I have discovered so far. The `canTransform` method gives the following error:
```camera 0 Lookup would require extrapolation into the future.  Requested time 68.824000000 but the latest data is at time 68.802956566, when looking up transform from frame [front_camera_link] to frame [local_origin]
camera 1 Lookup would require extrapolation into the future.  Requested time 68.824000000 but the latest data is at time 68.802956566, when looking up transform from frame [left_camera_link] to frame [local_origin]
camera 2 Lookup would require extrapolation into the future.  Requested time 68.824000000 but the latest data is at time 68.802956566, when looking up transform from frame [right_camera_link] to frame [local_origin]
```
The transformation is available only when the timestamp of the pointcloud matches the timestamp of the transformation.

Running the `tf_monitor` gives this result
```RESULTS: for local_origin to fcu
Chain is: local_origin -> fcu
Net delay     avg = 0.0640475: max = 0.0850794

Frames:
Frame: fcu published by unknown_publisher Average Delay: 0.0219803 Max Delay: 0.0655908

All Broadcasters:
Node: unknown_publisher 198.177 Hz, Average Delay: -0.00338286 Max Delay: 0.0655908
Node: unknown_publisher(static) 1e+08 Hz, Average Delay: 0 Max Delay: 0
```
As you can see in the frame view I have both the transformation from local_origin->fcu amd fcu->camera frame running at ~50Hz. While each point cloud is received at 20Hz.
[frames.pdf](https://github.com/PX4/avoidance/files/2496109/frames.pdf)

I went back to a older master (when the support for multiple pointcloud was not yet there) and it often happen that for one cycle the transformation is not available. Since it wasn't enough to show a timeout on the local_planner we never noticed.

**Comment by mrivi** [comment]
fixed in #122
